.github/workflows/pull-request-check.yml

```diff
@@ -29,4 +29,6 @@ jobs:
           overwrite-settings: false
 
       - name: Build and Test
-        run: mvn -B clean install
\ No newline at end of file
+        run: mvn -B clean install org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
```

Comment:
If you do CI based analysis you need to provide additionally
```xml
<sonar.organization>bci-oss</sonar.organization>`
<sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

Reply:
done